### DISCUSSION TITLE
Glean pretrained model

### DISCUSSION THREAD

**Discussion by Kyeonga-Kim** [open-mmlab/mmediting#1194] [Models]
Hi, I'm glad to refer the awesome architecture! :)
I'm wondering you can provide a pretrained model (16 -> 1024). 
I already know what you did with 16 inputs in the experiment. (I saw in the paper) If you have a pretrained model, can you share it?
Thx

**Comment by zengyh1900** [comment]
Hi @Kyeonga-Kim 
all the pre-trained models of Glean have been shown in https://github.com/open-mmlab/mmediting/tree/master/configs/restorers/glean 

I hope it helps!
